package/kernel/linux/modules/leds.mk

```diff
@@ -159,3 +159,19 @@ define KernelPackage/leds-uleds/description
 endef
 
 $(eval $(call KernelPackage,leds-uleds))
+
+define KernelPackage/leds-tlc591xx
+  SUBMENU:=$(LEDS_MENU)
+  TITLE:=LED driver for TLC59108 and TLC59116 controllers
+  DEPENDS:=+kmod-i2c-core +kmod-regmap-i2c
+  KCONFIG:= CONFIG_LEDS_TLC591XX
```

Comment:
Looks like a spurious space after that colon equal.

Reply:
Thanks!